src/cdk/v2/destinations/koala/procWorkflow.yaml

```diff
@@ -0,0 +1,67 @@
+bindings:
+  - name: EventType
+    path: ../../../../constants
+  - path: ../../bindings/jsontemplate
+  - name: defaultRequestConfig
+    path: ../../../../v0/util
+
+steps:
+  - name: validateInput
+    template: |
+      $.assert(.message.type, "message Type is not present. Aborting message");
+      $.assert(.message.type in {{$.EventType.([.IDENTIFY, .TRACK])}}, 
+        "message type " + .message.type + " is not supported");
+      $.assertConfig(.destination.Config.publicKey, "publicKey is not present. Aborting message");
+      $.context.email = .message.traits.email ?? .message.context.traits.email ?? .message.properties.email;
+      $.context.ko_profile_id = .message.traits.ko_profile_id ?? .message.context.traits.ko_profile_id ?? .message.properties.ko_profile_id;
+      $.assert($.context.email || $.context.ko_profile_id, "Neither email or ko_profile_id are present on traits. Aborting message");
+  - name: setMessageType
+    template: |
+      $.context.messageType = .message.type.toLowerCase();
+  - name: preparePayloadForIdentify
+    condition: $.context.messageType === {{$.EventType.IDENTIFY}}
+    template: |
+      const traits = .message.traits ?? .message.context.traits ?? {};
+      const koTraits = traits{~['ko_profile_id']}
+      const basePayload = {
+        email: .message.().({{{{$.getGenericPaths("email")}}}}),
+        profile_id: traits.ko_profile_id,
+        identifies: [{
+          type: $.context.messageType,
+          event: .message.event, 
```

Comment:
```suggestion
```

Reply:
Identify call don't have `event` property. Also, from koala doc [here](https://getkoala.com/docs/developer-guides/server-side)
event is not present inside `identifies`.
If needed we can assign $.context.messageType to `event`